spec/kittikrb/cursor_spec.rb

```diff
@@ -122,15 +122,219 @@ def current_rendered_buffer
+
+      describe '#erase' do
+        it 'should erase(write whitespace) the specified region in TTY' do
+          expect($stdout).to receive(:write).exactly(6).times.with("\e\s")
+          expect(cursor.erase(1, 1, 3, 2)).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToEnd' do
+        it 'should erase(write whitespace) all chars to end current row ' \
+           'in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(width - 6 + 1).times
+            .with("\e\s")
+          expect(cursor.eraseToEnd).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToStart' do
+        it 'should erase(write whitespace) all chars to current position in ' \
+           'row in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(6).times.with("\e\s")
+          expect(cursor.eraseToStart).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToDown' do
+        it 'should erase(write whitespace) all chars to current position in ' \
+           'column in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(6 + 1).times.with("\e\s")
+          expect(cursor.eraseToDown).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseToUp' do
+        it 'should erase(write whitespace) all chars to end current column ' \
+           'in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(height - 6).times
+            .with("\e\s")
+          expect(cursor.eraseToUp).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseLine' do
+        it 'should erase(write whitespace) all chars in current row in TTY' do
+          cursor.move_to 5, 6
+
+          expect($stdout).to receive(:write).exactly(width).times.with("\e\s")
+          expect(cursor.eraseLine).to be_instance_of described_class
+        end
+      end
+
+      describe '#eraseScreen' do
+        it 'should erase(write whitespace) all chars in TTY' do
+          expect($stdout).to receive(:write).exactly(width * height).times
+            .with("\e\s")
+          expect(cursor.eraseScreen).to be_instance_of described_class
+        end
+      end
+    end
+
+    context 'cursor methods' do
+      describe '#foreground' do
+        it 'should properly set foreground color' do
+          expect(cursor.foreground(8)).to be_instance_of described_class
+          expect(cursor.instance_variable_get(:@foreground)).to eq 8
+        end
+      end
+
+      describe '#background' do
+        it 'should properly set background color' do
+          expect(cursor.background(5)).to be_instance_of described_class
+          expect(cursor.instance_variable_get(:@background)).to eq 5
+        end
+      end
+
+      describe '#bold' do
+        it 'should properly set display bold parameter' do
```

Comment:
`should properly toggle bold mode`

https://github.com/kittikrb/cursor/pull/4/files#r55125092